lib/jpeg.xml

```diff
@@ -0,0 +1,149 @@
+<?xml version="1.0" ?>
+<?xml-stylesheet type='text/xsl' href='interface.xsl'?>
+<interface uri="http://repo.roscidus.com/lib/jpeg" xmlns="http://zero-install.sourceforge.net/2004/injector/interface" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://zero-install.sourceforge.net/2004/injector/interface http://0install.de/schema/injector/interface/interface.xsd http://0install.de/schema/desktop-integration/capabilities http://0install.de/schema/desktop-integration/capabilities/capabilities.xsd">
+  <name>Jpeg</name>
+  <summary xml:lang="en">Jpeg: library and tools for JPEG images</summary>
+  <description xml:lang="en">This package contains C software to implement JPEG image compression and decompression. JPEG is a standardized compression method for full-color and gray-scale images. JPEG is intended for &quot;real-world&quot; scenes; cartoons and other non-realistic images are not its strong suit. JPEG is lossy, meaning that the output image is not identical to the input image. The user can trade off output image quality against compressed file size by adjusting a compression parameter. 
+
+The distributed programs provide conversion between JPEG &quot;JFIF&quot; format and image files in PBMPLUS PPM/PGM, GIF, BMP, and Targa file formats. The core compression and decompression library can easily be reused in other programs, such as image viewers. 
+
+Includes EXIF patches from: 
+http://bugzilla.redhat.com/bugzilla/show_bug.cgi?id=106060 http://sylvana.net/jpegcrop/exifpatch.html 
+</description>
+  <icon href="https://raw.githubusercontent.com/0install/repo.roscidus.com/master/lib/jpeg.ico" type="image/vnd.microsoft.icon"/>
+  <icon href="https://raw.githubusercontent.com/0install/repo.roscidus.com/master/lib/jpeg.ico" type="image/png"/>
+  <category>Graphics</category>
+  <homepage>https://web.archive.org/web/20171208102837/http://gnuwin32.sourceforge.net/packages/jpeg.htm</homepage>
```

Comment:
http://gnuwin32.sourceforge.net/packages/jpeg.htm seems to be up, so maybe it's better to link there directly rather than using WaybackMachine.

Reply:
odd must have been temporarily down when I first packaged it
done